### ISSUE TITLE
More serializer types

### ISSUE THREAD

**Issue by oberstet** [wamp-proto/wamp-proto#355]
Crossbar.io supports the following serializers: `[u'json', u'msgpack', u'cbor', u'ubjson', u'flatbuffers']` using the following serializer IDs (the string is for use with websocket transport, whereas the integers are [for rawssocket](https://wamp-proto.org/_static/gen/wamp_latest_ietf.html#handshake)):

* `json`: 1
* `msgpack`: 2
* `cbor`: 3
* `ubjson`: 4
* `flatbuffers`: 5

**Comment by konsultaner** [comment]
@oberstet Have you ever had a look at https://github.com/arangodb/velocypack?

**Comment by oberstet** [comment]
No, I haven't. fwiw, we're mostly using one of cbor or json, and begin to use flatbuffers these days depending on contexts.

**Comment by oberstet** [comment]
fwiw, here is a performance comparison of the different serializers (crossbar/autobahnpy): https://crossbario.com/docs/benchmarks/serialization/index.html

**Comment by KSDaemon** [comment]
I don't see any reason why not to add this serializers to spec to [Advanced Transports and Serializers](https://wamp-proto.org/wamp_latest_ietf.html#name-advanced-transports-and-ser) section.
